akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterShardingSettings.scala

```diff
@@ -311,6 +311,11 @@ final class ClusterShardingSettings(
   private[akka] def shouldHostShard(cluster: Cluster): Boolean =
     role.forall(cluster.selfMember.roles.contains)
 
+  /** If true, idle entities should be passivated if they have not received any message by this interval, otherwise it is not enabled. */
+  @InternalApi
+  private[akka] def shouldPassivateIdleEntities: Boolean =
+    passivateIdleEntityAfter > Duration.Zero && !rememberEntities
+
```

Comment:
Not naming `shouldPassivateIdleEntity` because this takes `rememberEntities` and `settings.passivateIdleEntityAfter` into consideration.

Reply:
Could it be a val even?

Reply:
YES how did I miss, thanks.